Form-Controls/index.html

```diff
@@ -14,14 +14,51 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div> 
+      <label for="name">Name</label>
+      <input type="text" id="name" placeholder="Enter Your Name" pattern="^[a-zA-Z]+(?:[-' ][a-zA-Z]+)*$" required>
+    </div>
+    <div> 
+      <label for="email">Email</label>
+      <input type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" id="email" placeholder="Enter Your e-mail" required>
+    </div>
+    <div>
+    Choose your T-shirt's colour
+        <div>
+            <label for="Red">Red</label>
+            <input type="radio" name="colour" id="Red" value="Red" required>
```

Comment:
When it comes to HTML attributes and conventions around naming, it's generally better to use lower case, as opposed to having Capitalisation in your attributes. 

So generally `id="size"` and `name="size"` will improve consistency, readability and will follow HTML conventions, which will make other developers love your work ethic 🥳